src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/azuremonitorprofile.py

```diff
@@ -50,6 +50,7 @@ def link_azure_monitor_profile_artifacts(
 
 # pylint: disable=line-too-long
 def unlink_azure_monitor_profile_artifacts(cmd, cluster_subscription, cluster_resource_group_name, cluster_name):
+    print("Executing this action will result in the removal of all custom resources for the `azmonitoring.coreos.com` custom resource defintion. Are you certain that you wish to proceed with this operation?")
```

Comment:
Have you considered using logger.warning("xxx")? Compared with print(), log will be printed in color (if shell supports) to stderr.

Reply:
Thank you for the suggestion. I have updated the PR to reflect this.